accounts-tool/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "solana-accounts-tool"
+version = "1.15.0"
+description = "Solana Accounts Tool"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+repository = "https://github.com/solana-labs/solana"
+homepage = "https://solana.com/"
+documentation = "https://docs.rs/solana-accounts-tool"
+license = "Apache-2.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
I'm of the opinion that we should rip all of the dependencies out until we need them. The idea would be starting this brand new crate with a "clean slate" instead of dragging everything over from the `'ledger-tool` Cargo.toml